### DISCUSSION TITLE
103 LTS  Announcement

### DISCUSSION THREAD

**Discussion by richtja** [avocado-framework/avocado#5729] [Announcement]
Hi Avocado community,

a few weeks ago we released Avocado 102 and we started work on 103 LTS
(Long Term Stability). The release 103 will consist strictly of bug
fixes, usability, and documentation updates as a preparation for LTS.
Therefore, we would like to ask everyone here to try version 102 (If you
have done it yet) and let us know about any issues and concerns which
you might have. Such testing will be very helpful for us and it makes
upgrade to new LTS version easier for community.

Because of the preparations for LTS, we would like to ask maintainers to
avoid merging changes that do not fall into these categories:
 * Bug fixes
 * Usability Improvements
 * Documentation updates

As for the Avocado utility modules (“avocado.utils”) it is OK to
introduce new functionality, but changes to the existing APIs (including
interface and behavior) should be avoided.  This kind of change should
wait until sprint 104.

Thank you everyone and Happy hacking and testing!

**Comment by nirGNU** [comment]
HI @richtja,
Can you please help with a tentative timeline for this LTS? 
According to the last [discussion](https://github.com/avocado-framework/avocado/discussions/5711#discussioncomment-6259534)  it  was planned to be release in July-Aug. 
Is there any change in the dates?

Thank you,
-Nir

**Reply by richtja** [reply]
Hi @nirGNU sorry for the late reply. Unfortunately, the LTS release has been delayed, because of personal changes in our team and number of revealed bugs, which have to be solved before LTS. We had to reconsider our planning and now we estimated the release to the middle of October. I hope this helps.

**Reply by nirGNU** [reply]
Hi @richtja , 

When can I expect this release? 
Our release is tied up with this LTS. 

Thank you,
-Nir

**Reply by richtja** [reply]
Hi @nirGNU, unfortunately we are still not prepared for the release. We are still working on a [bug fixes ](https://github.com/orgs/avocado-framework/projects/1/views/3?sortedBy%5Bdirection%5D=desc&sortedBy%5BcolumnId%5D=Status)which needs to be done before the release. I am so sorry for problems which this creates on your side, but we will need minimally one more month to solve all the known issues.

**Reply by nirGNU** [reply]
Hi @richtja,

Do you have any update on the LTS support?
I am looking forward for this release by Jan first week, can you please help with your timeline?

Awaiting your reply.

Thank you,
-Nir.

**Reply by nirGNU** [reply]
Hi @richtja and @clebergnu,

A gentle reminder.

Thank you,
-Nir.

**Reply by richtja** [reply]
Hi @nirGNU, I am so sorry that you have to wait for the release for so long, but we should get the release done soon. If you look at our [project board](https://github.com/orgs/avocado-framework/projects/1/views/2), you can see that we are almost done. We have only one bug #5668 and one PR #5621 missing, we almost finished documentation review and right now we are working on testing and release notes. Because of this status, we would like to do the release during the January 2024. I hope this info will help you, and I am sorry again for taking it so long.

**Reply by nirGNU** [reply]
Hi @richtja, Just got a notification of the 103 LTS release and I am so pleased that you provided this early in January. 
We are going through the change history now it will help us to see how our plugins will behave.
I appreciate your support in considering latest code for LTS.

Thank you. :)

**Comment by richtja** [comment]
The [103 LTS](https://avocado-framework.readthedocs.io/en/latest/releases/lts/103_0.html) has beet released. Thank you all for participation.
